### ISSUE TITLE
Electrum does not start anymore after upgrading to Ubuntu 24.04.1 LTS

### ISSUE THREAD

**Issue by BTCBellyButton** [spesmilo/electrum#9213]
Trying to run Electrum after upgradig to Ubuntu 24.04.1 LTS returns this:

Error: No module named 'qrcode'. Try 'sudo python3 -m pip install <module-name>'

See also:  #9212

**Comment by DireWolfM14** [comment]
Try reinstalling Electrum by following these instructions:
https://bitcointalk.org/index.php?topic=5456909.msg62430610#msg62430610

**Comment by BTCBellyButton** [comment]
Thanks!
I reinstalled it by using the AppImage.

**Comment by SomberNight** [comment]
I guess more details would be needed re exactly how electrum was installed, and how the dependencies such as `qrcode` were installed. But if both were installed before upgrading to 24.04, and then the ubuntu upgrade was done, I believe Electrum should still work. (the `externally-managed-environment` changes re https://github.com/spesmilo/electrum/issues/8328 should only restrict/affect newly installed packages)

**Comment by BTCBellyButton** [comment]
This: "both were installed before upgrading to 24.04, and then the ubuntu upgrade was done".

I got the `error: externally-managed-environment`  when trying to re-install it so I resorted to the AppImage.

I had this problem with Specter too just after upgrading to Ubuntu 24.04 so I had to reinstall that within a new virtual environment. 

No big deal but others could find themselves in a similar situation by following the installation instructions on the website, so it'd be helpful to review them. 
Maybe like @DireWolfM14 suggested?
